dockers/docker-dhcp-relay/Dockerfile.j2

```diff
@@ -50,14 +54,14 @@ COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
 COPY ["critical_processes", "/etc/supervisor"]
 COPY ["cli", "/cli/"]
 
-{% if include_system_eventd == "y" and build_reduce_image_size != "y" %}
+RUN mkdir -p /etc/rsyslog_plugin
+
 # Copy regex json and rsyslog_plugin.conf file into rsyslog.d
-COPY ["*.json", "/etc/rsyslog.d/"]
-COPY ["files/rsyslog_plugin.conf.j2", "/etc/rsyslog.d/"]
+COPY ["*.json", "/etc/rsyslog_plugin/"]
+COPY ["files/rsyslog_plugin.conf.j2", "/etc/rsyslog_plugin/"]
 # Create dhcp_relay_regex.conf
-RUN j2 -f json /etc/rsyslog.d/rsyslog_plugin.conf.j2 /etc/rsyslog.d/events_info.json  > /etc/rsyslog.d/dhcp_relay_events.conf
-RUN rm -f /etc/rsyslog.d/rsyslog_plugin.conf.j2
-RUN rm -f /etc/rsyslog.d/events_info.json
-{% endif %}
+RUN j2 -f json /etc/rsyslog_plugin/rsyslog_plugin.conf.j2 /etc/rsyslog_plugin/events_info.json  > /etc/rsyslog_plugin/dhcp_relay_events.conf
```

Comment:
>j2 [](http://example.com/codeflow?start=4&length=2)

we should move j2 into start.sh, and also use include_system_eventd into the j2 itself. so no need to conditional copy inside start.sh

Reply:
we normally just COPY j2 files into /usr/share/sonic/templates/ inside containers.

Reply:
Done